### DISCUSSION TITLE
下载文件没有速度，一直卡在0%，本地的网络是没有问题的。

### DISCUSSION THREAD

**Discussion by cmdragon** [messense/aliyundrive-webdav#354] [Q&A] [answered]
### 问题描述

![image](https://user-images.githubusercontent.com/39700024/161183066-d7241e75-8a74-4add-ad3f-7bbfccfd9644.png)
挂了一晚上没有下载速度，上传没什么问题。就行下载文件的时候会没有速度，一直显示正在下载0%。

### 重现步骤

上传文件测试一点问题没有

### 版本

最新版

### 运行平台

docker

### 日志

.

**Comment by messense** [comment, marked as answer]
请提供日志

**Reply by cmdragon** [reply]
![image](https://user-images.githubusercontent.com/39700024/161186884-97e0276c-1e15-488c-842e-49b6b71ae2e3.png)

**Reply by cmdragon** [reply]
还需要提供什么吗，不知道是什么问题造成429. 我刚启动就报的这个错误，并且我远程文件夹里面只有这一个视频。

**Reply by messense** [reply]
https://github.com/messense/aliyundrive-webdav/discussions/326
